### PR TITLE
Added dpkg files.

### PR DESCRIPTION
**Pull request by joachimmetz**
@destijl for review. Not urgent adding dpkg packaging files in this CL.

### CODE REVIEW

File: config/dpkg/control [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Source: python-artifacts
+Section: python
+Priority: extra
+Maintainer: Log2Timeline <log2timeline-dev@googlegroups.com>
```
Comment (destijl) [inline config/dpkg/control]: whoops

File: config/dpkg/copyright [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Format: http://dep.debian.net/deps/dep5
+Upstream-Name: artifacts
+Source: https://github.com/log2timeline/artifacts/
```
Comment (destijl) [inline config/dpkg/copyright]: here too

**Comment by joachimmetz** [comment]
Good catch. I've fixed them. I've also added a URL regarding SystemDrive and some textual corrections.

File: definitions/windows_bootstrap.yaml [hunk 3]
```diff
@@ -60,14 +60,16 @@ urls: ['http://environmentvariables.org/ProgramFiles']
 name: SystemDriveEnvironmentVariable
 doc: |
   The %SystemDrive% environment variable, usually "C:".
```
Comment (destijl) [inline definitions/windows_bootstrap.yaml]: style guide says we should have a blank line here, I forgot that yesterday.

**Comment by destijl** [comment]
LGTM, one minor comment.

**Comment by joachimmetz** [comment]
Ack, documentation string fixed. I'll merge once Travis says OK.

**Pull request merged**